tests/test_notifications.py

```diff
@@ -462,10 +462,13 @@ def test_get_configured_project_ids_includes_top_level_private_projects_if_subsc
         assert_in(private_project._id, configured_project_ids)
 
     def test_get_configured_project_ids_excludes_private_projects_if_no_subscriptions_on_node(self):
+        from website.project.mailing_list import create_mailing_list_subscription
+        node_created.disconnect(create_mailing_list_subscription)
         private_project = factories.ProjectFactory()
         node = factories.NodeFactory(parent=private_project)
         configured_project_ids = utils.get_configured_projects(node.creator)
         assert_not_in(private_project._id, configured_project_ids)
+        node_created.connect(create_mailing_list_subscription)
```

Comment:
This should be in a `try/finally` to avoid leaving the test suite in an inconsistent state if there's a failure in between `disconnect` and `connect`—or moved to `setup`/`teardown`.